packages/project-access/src/index.ts

```diff
@@ -47,7 +47,13 @@ export {
     readUi5Yaml,
     refreshSpecificationDistTags,
     toReferenceUri,
-    updatePackageScript
+    updatePackageScript,
+    hasCdsPluginUi5,
+    getWorkspaceInfo,
+    getWorkspacePackages,
+    minCdsVersion,
```

Comment:
Perhaps we add this to `constants.ts` and change it to Uppercase? Also a more speakable name might help, like `MinCdsVersionUi5Plugin`?